### ISSUE TITLE
Docking branch

### ISSUE THREAD

**Issue by dri-richard** [britzl/extension-imgui#36]
imgui has a long-lived docking branch, and although it doesn't look like it's going to be merged anytime soon, it is considered stable - it's mentioned in release notes, and tags are created for it.

It would be useful for extension-imgui to also have a docking branch that can exploit those new capabilities, especially in the context of netimgui, as it would expand the types of 'apps' possible.

**Comment by britzl** [comment]
Yes, this might be useful to have. I am a bit concerned about the added work maintaining two branches and making sure to merge fixes from the main branch to the docking branch.
